tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,53 @@
+- name: Test agent configuration
+  hosts: wazuh-windows
+  tasks:
+
+    - name: Create temp folder
+      win_file:
+        path: C:\temp
+        state: directory
+
+    - name: Copy ossec.conf
+      ansible.windows.win_copy:
+        src: C:\Program Files (x86)\ossec-agent\ossec.conf
+        dest: C:\temp
+        remote_src: true
+
+    - name: Enable the agent to collect eventchannel logs
```

Comment:
Specify Windows Defender

Reply:
Done in dfea381fb5e5080842a3344bb57f7b3860228f21